CONTRIBUTING.md

```diff
@@ -64,12 +64,6 @@ Now that you're a part of the GitHub Docs community, you can keep participating
   - [Labels](#labels)
 - [Opening a pull request](#opening-a-pull-request)
 - [Working in the github/docs repository](#working-in-the-githubdocs-repository)
-- [Resolving merge conflicts](#resolving-merge-conflicts)
-  - [In the GitHub user interface](#in-the-github-user-interface)
-    - [Editing the file and committing the changes](#editing-the-file-and-committing-the-changes)
-- [Troubleshooting](#troubleshooting)
-  - [Failed status checks](#failed-status-checks)
-- [Draft pull requests](#draft-pull-requests)
```

Comment:
I didn't find any content to link these topics to. So ended up removing them as suggested by [Janiceilene](https://github.com/janiceilene).